cmd/kola/kola.go

```diff
@@ -250,12 +269,50 @@ func (i item) String() string {
 			}
 		}
 		i.Platforms = platforms
+	} else {
+		i.ExcludePlatforms = []string{""}
 	}
 	if len(i.Platforms) == 0 {
-		i.Platforms = []string{"all"}
+		if listJSON {
+			i.Platforms = kolaPlatforms
+		} else {
+			i.Platforms = []string{"all"}
+		}
 	}
 	if len(i.Architectures) == 0 {
-		i.Architectures = []string{"all"}
+		if listJSON {
+			i.Architectures = kolaArchitectures
+		} else {
+			i.Architectures = []string{"all"}
+		}
 	}
-	return fmt.Sprintf("%v\t%v\t%v", i.Name, i.Platforms, i.Architectures)
+	if len(i.ExcludeDistros) > 0 {
+		excludeDistros := map[string]struct{}{}
+		for _, distro := range i.ExcludeDistros {
+			excludeDistros[distro] = struct{}{}
+		}
+		if len(i.Distros) == 0 {
+			i.Distros = kolaDistros
+		}
+		distros := []string{}
+		for _, distro := range i.Distros {
+			if _, ok := excludeDistros[distro]; !ok {
+				distros = append(distros, distro)
+			}
+		}
+		i.Distros = distros
+	} else {
+		i.ExcludeDistros = []string{""}
+	}
+	if len(i.Distros) == 0 {
```

Comment:
This block could also be abstracted into a helper, though there seems less value in it.